modules/cas_cache/layer_cache_management.c

```diff
@@ -777,6 +777,12 @@ int cache_mngt_purge_object(const char *cache_name, size_t cache_name_len,
 	if (result)
 		return result;
 
+	if (ocf_cache_is_passive(cache))
```

Comment:
this should probably be checked under cache read lock (acquired in line 786 below)